modules/oe_webtools_media/tests/src/FunctionalJavascript/MediaSourceWebtoolsTest.php

```diff
@@ -75,7 +76,7 @@ public function testMediaWebtoolsSource(): void {
       // Create a webtools media item with invalid webtools snippet.
       $this->drupalGet("media/add/{$media_type_id}");
       $assert_session->fieldExists('Name')->setValue("Invalid webtools $widget_name item");
-      $assert_session->fieldExists("Webtools {$widget_name} snippet")->setValue('{"service": "' . $service . $widget_type . '"}');
+      $assert_session->fieldExists("Webtools {$widget_name} snippet")->setValue('{"service": "' . $invalid_service . '"}');
```

Comment:
After adapting the validation messages we should cover both cases of invalid or blacklisted services.